drivers/input/touchscreen/synaptics_dsx_i2c.c

```diff
@@ -3809,11 +3857,14 @@ static void synaptics_rmi4_set_params(struct synaptics_rmi4_data *rmi4_data)
 	set_bit(KEY_HOMEPAGE, rmi4_data->input_dev->keybit);
 	set_bit(KEY_F3, rmi4_data->input_dev->keybit);
 	set_bit(KEY_DOUBLE_TAP, rmi4_data->input_dev->keybit);
```

Comment:
MCE can/does handle power key as double tap, but I was thinking more in the line of: "After the changes elsewhere, is the kernel driver ever going to emit power key events anymore?" If not (i.e. KEY_DOUBLE_TAP aka KEY_POWER is always converted to MSC_GESTURE event), then it would be incorrect to claim that the input device can emit power key events.